portfolio.html

```diff
@@ -234,6 +234,23 @@ <h2  class="listHeading">Tutoring Assistant</h2>
         <p id="Projects" class="header">Projects</p>
 
         <ul>
+            <li>
+                <h2  class="listHeading">TransitPal</h2>
```

Comment:
We should remove the whitespace here

```suggestion
                <h2 class="listHeading">TransitPal</h2>
```